starters/next-wordpress-starter/components/grid.js

```diff
@@ -0,0 +1,58 @@
+import Link from "next/link";
+import Image from "next/image";
+
+export const withGrid = (Component) => {
+  const GridedComponent = ({ data, ...props }) => {
+    if (!data || !data.length) {
+      return props.contentType ? (
+        <h2 className="text-xl text-center mt-14">
+          No {props.contentType} found 🏜
+        </h2>
+      ) : null;
+    }
+
+    return (
+      <div
+        className={`mt-12 grid gap-5 max-w-content mx-auto lg:grid-cols-3 lg:max-w-screen-lg`}
+      >
+        {data.map((content, i) => {
+          return <Component key={i} content={content} {...props} />;
+        })}
+      </div>
+    );
+  };
+
+  return GridedComponent;
+};
+
+export const PostGridItem = ({ content: post }) => {
+  const imgSrc = post?.featuredImage?.node.sourceUrl || "";
+  return (
+    <Link passHref href={post.uri}>
+      <a>
+        <div className="flex flex-col rounded-lg shadow-lg overflow-hidden cursor-pointer border-2 h-full hover:border-indigo-500">
+          <div className="flex-shrink-0 relative h-40">
+            {imgSrc !== "" ? (
+              <Image
+                src={imgSrc}
+                layout="fill"
+                objectFit="cover"
+                alt={post.title}
```

Comment:
Since there is `altText` available on the `featuredImage` node, let's use it here.